Formula/ksync.rb

```diff
@@ -0,0 +1,27 @@
+class Ksync < Formula
+  desc "Sync files between your local system and a kubernetes cluster"
+  homepage "https://ksync.github.io/ksync/"
+  url "https://github.com/ksync/ksync.git",
+    :tag  => "0.4.5"
+  sha256 "bd47c5b5418bfc6d2f08abe855a52d4189f8bf380747cdd4e5c1a3ca0a87498a"
+  license "Apache-2.0"
+
+  depends_on "go" => :build
+
+  def install
+    # Set some stamps and build local CLI binary
+    time = `date +%Y%m%dT%H%M%S`.chomp
+    goversion = `go version | awk '{ print $3 }'`
```

Comment:
I think something like this would probably be more preferable:
```
goversion = "go#{Formula["go"].version.to_s}"

Reply:
Readability?

Reply:
That's a fair point. I'll leave the call to a different maintainer, although I'll put the option out here.

Reply:
Happy to do it, just want to match convention and be consistent with the binary that is distributed via non-homebrew methods.

Reply:
I think the suggested improvement is more in line with homebrew state of the art.